README.md

```diff
@@ -6,13 +6,13 @@
 Static server that hosts Firefox Account sign up, sign in, email verification, etc. flows.
 
 ## Prerequisites
-
+Follow the instructions on :
+	https://github.com/mozilla/fxa-local-dev
+The above link helps to install the following pre-requisites, and set up a development environment.
 * node 0.10.x
 * npm
-* Grunt (`npm install -g grunt-cli`)
+* Grunt
 * libgmp
-  * On Linux, Install libgmp and libgmp-dev packages: `sudo apt-get install libgmp3-dev`
-  * On Mac OS X: `brew install gmp`
 * [fxa-auth-server](https://github.com/mozilla/fxa-auth-server) running on 127.0.0.1:9000.
 
 ## Development Setup
```

Comment:
it might make sense to add fxa-local-dev here instead. and mention that fxa-local-dev is a better way to contribute to Firefox accounts.